homeassistant/components/logbook/__init__.py

```diff
@@ -331,7 +333,6 @@ def humanify(
     """Generate a converted list of events into Entry objects.
 
     Will try to group events if possible:
-    - if 2+ sensor updates in GROUP_BY_MINUTES, show last
```

Comment:
The SQL query generated by `_get_events` already filters out all entities from the `sensor` and `proximity` domains which have a unit of measurement; hence the grouping here filled no purpose and is removed in this PR.